choco/chocolateyUninstall.ps1

```diff
@@ -13,8 +13,9 @@ if ($installerType -ne 'MSI') {
 
 $uninstalled = $false
 $local_key     = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*'
-$machine_key   = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\*'
+$machine_key   = 'HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*'
 $machine_key6432 = 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\*'
```

Comment:
Why are the `uninstalled`, `local_key`, `machine_key` and `key` variables used? I can't see them being used anywhere (at least in this file)?